tests/test_onnx_v2.py

```diff
@@ -243,6 +315,29 @@ def _pytorch_export(self, test_name, name, model_name, feature, onnx_config_clas
                 self.fail(f"{name}, {feature} -> {e}")
 
     @parameterized.expand(_get_models_to_test(PYTORCH_EXPORT_MODELS))
```

Comment:
Should this be:

```suggestion
    @parameterized.expand(_get_models_to_test(TENSORFLOW_EXPORT_MODELS))
```

I'm also wondering if there's anything intrinsically PyTorch specific about the `_pytorch_export()` function? Can't we reuse this for the TensorFlow tests?

Reply:
We can. Thank you! :)